src/main/kotlin/io/gitlab/arturbosch/detekt/sonar/sensor/IssueReporter.kt

```diff
@@ -48,8 +47,11 @@ class IssueReporter(
                     return null
                 }
             }
-            BaselineFacade(baselinePath.toPath())
+            return BaselineFacade(baselinePath.toPath())
         }
+        return config.get(BASELINE_KEY)
+            .orElse(null)
+            ?.let(::createBaselineFacade)
```

Comment:
Wouldn't it be more logical to reorder the statements?

```suggestion
        return config.get(BASELINE_KEY)
            ?.let(::createBaselineFacade)
            .orElse(null)
```